THIRD PARTY NOTICES

```diff
@@ -0,0 +1,19 @@
+This file is based on or incorporates material from the projects listed below (Third Party OSS). The original copyright notice and the license under which Microsoft received such Third Party OSS, are set forth below. Such licenses and notices are provided for informational purposes only. Microsoft licenses the Third Party OSS to you under the licensing terms for the Microsoft product or service. Microsoft 
```

Comment:
Never read this before lol.
@MIchaelMainer  @roinochieng  Do these need to be updated regularly as we introduce more dependencies?

Reply:
The guidance on this has certainly been updated. https://docs.opensource.microsoft.com/tools/cg/features/notice/

It looks like third party notice is no longer the way to go. https://docs.opensource.microsoft.com/tools/tpn/